crates/net/network/src/transactions/mod.rs

```diff
@@ -75,13 +75,16 @@ pub use validation::*;
 const PEER_TRANSACTION_CACHE_LIMIT: usize = 1024 * 10;
 
 /// Soft limit for NewPooledTransactions
-const NEW_POOLED_TRANSACTION_HASHES_SOFT_LIMIT: usize = 4096;
+const NEW_POOLED_TRANSACTIONS_BROADCAST_PACKET_HASHES_COUNT_SOFT_LIMIT: usize = 4096;
 
-/// Soft limit for the response size of a GetPooledTransactions message (2MB) in bytes. Standard
-/// maximum response size. See specs
+/// Soft limit for the message of full transactions in bytes.
```

Comment:
this is outgoing announcement, right?

Reply:
no this is originally used by the `FullTransactionsBuilder`